routing/missioncontrol_test.go

```diff
@@ -204,12 +204,12 @@ func TestMissionControlChannelUpdate(t *testing.T) {
 	ctx.reportFailure(
 		0, lnwire.NewFeeInsufficient(0, lnwire.ChannelUpdate{}),
 	)
-	ctx.expectP(0, testAprioriHopProbability)
+	ctx.expectP(100, testAprioriHopProbability)
```

Comment:
Why has this value changed?

Reply:
Amount `0` was never supported, but with the changes in this commit it returns a different result.

I have been thinking about adding an `error` return value to catch this case, but not sure if it is worth it.

Reply:
Should be fine as is.